justfile

```diff
@@ -152,3 +152,22 @@ quick-modify:
 # just show-logs | bunyan
 show-logs:
     tail -f $(ls $PGT_LOG_PATH/server.log.* | sort -t- -k2,2 -k3,3 -k4,4 | tail -n 1)
+
+agentic name:
+    unset ANTHROPIC_API_KEY && claude --dangerously-skip-permissions -p "please read agentic/{{name}}.md and follow the instructions closely"
```

Comment:
not sure we should commit this into an open source repo – seems like a vulnerability for prompt injection :)
We could pass a --settings `agentic/{{name}}.settings.json` file ? 

Also, I'm curious: Why do you unset the API key here? To force somebody to login?

Reply:
good point, I just commented them out for now with a node 'use at your own risk'